Photon-Tinker/Info.plist

```diff
@@ -48,9 +48,13 @@
 		<true/>
 	</dict>
 	<key>NSBluetoothPeripheralUsageDescription</key>
-	<string>No usage</string>
+	<string>Particle app uses bluetooth to communicate with 3rg generation hardware.</string>
```

Comment:
A few small things:

1. "Bluetooth" is a proper noun, so it's supposed to always be capitalized
2. **3rg** generation is the best generation.
3. I'd ditch "conveniently" from "conveniently scan stickers" -- "ease the setup process" already implies convenience.
4. (Just going to rewrite this last line with the needed corrections instead of describing it): "Particle app requires location services to read the SSID of the Wi-Fi network your device is currently connected to."

All of these are small details, but I wasn't sure if these are seen by the public or just reviewers